Update package manifest and property editor docs (#5795)
umbraco/UmbracoDocs: 10/umbraco-cms/extending/property-editors/package-manifest.md

```json
{
    "name": "Suggestions",
    "version": "1.0.0 beta",
    "allowPackageTelemetry": true,
    "bundleOptions": "Default",
    "packageView": "/App_Plugins/Suggestions/suggestion-config.html",
    "propertyEditors": [
        {

>             "alias": "Suggestions",

Author: not necessarily. It works with both small and big first letter.

            "name": "Suggestions",
            "editor": {
                "view": "/App_Plugins/Suggestions/suggestion.html",
                "hideLabel": true,
                "valueType": "JSON",
                "supportsReadOnly": true
            }
        }
    ],
    "javascript": [
        "/App_Plugins/Suggestions/suggestion.controller.js"
    ]
}
```

## Sample Manifest with Csharp

You can also register your files by implementing a `IManifestFilter` instead of creating a `package.manifest`. Create a `ManifestFilter.cs` file and implement the `IManifestFilter` interface. Then define the composer using the `IComposer` interface.

```csharp

> namespace MyProject

Reviewer: using Umbraco.Cms.Core.Composing;

Author: there are 2 more using statements apart from composing, I will add them as well

### Telemetry elements

> From version 9.2 some additional root elements was added, the purpose of these are to control and facilitate telemetry about the package, none of these are mandatory. The properties are:

Reviewer 2: ```suggestion
From version 9.2 additional root elements were added, the purpose of these is to control and facilitate telemetry about the package, none of these are mandatory. The properties are:
```
